static/solo-cta/index.js

```diff
@@ -206,13 +206,19 @@ window[cbName] = function (data) {
       }
       
       overlayEl.innerHTML = template;
-
+      var configCopy = JSON.parse(JSON.stringify(config));
+      var onPlayerChange = "onPlayerChange" in config ? 'undefined' !== typeof config.onPlayerChange ? config.onPlayerChange :null : null;
```

Comment:
I think you can skip this check:

`var onPlayerChange = "onPlayerChange" in config ? 'undefined' !== typeof config.onPlayerChange ? config.onPlayerChange :null : null;`

Just let the `configCopy` pass and let the _Player_ singleton do the checks in this line:

`this.onPlayerChange = isset(options.onPlayerChange) ? options.onPlayerChange : null;`

This should be enough.
The less code the better.